### PR TITLE
port InfoPanel/InfoPanelColumn components

### PR DESCRIPTION
**Pull request by natebrennand**
**Overview:**

Simply porting the `InfoPanel` & `InfoPanelColumn` components in from the apps-dashboard.

**Screenshots/GIFs:**

 screenshot 2017-03-21 17 15 38

**Testing:**
- [x] Unit tests
- Manual tests:
  - [x] Chrome
  - [ ] Safari
  - [ ] IE10

**Roll Out:**
- Before merging:
  - [x] Updated docs
- After merging:
  - [ ] Deployed updated docs (`make deploy-docs`)

### CODE REVIEW

**Comment by natebrennand** [comment]
CC @CarlNelson

**Comment by natebrennand** [comment]
example: screenshot 2017-03-21 17 15 38

File: src/InfoPanel/InfoPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import "./InfoPanel.less";
+
+
+/**
+ * Base presentational component for the District Profile info panels.
```
Comment (arsalansufi) [inline src/InfoPanel/InfoPanel.jsx]: I'd change this comment given that the we'll now be using `InfoPanel`s for purposes other than the District Profile page

File: src/InfoPanel/InfoPanel.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.InfoPanel {
```
Comment (arsalansufi) [inline src/InfoPanel/InfoPanel.css]: Should we be committing this css file? We already have the less file and no other components in the `src` directory seem to include css files.

Comment (natebrennand) [inline src/InfoPanel/InfoPanel.css]: 👍  had just copied the whole directory from apps-dashboard wholesale

**Pull request merged**